dpctl-capi/tests/test_sycl_device_interface.cpp

```diff
@@ -26,6 +26,7 @@
 
 #include "dpctl_sycl_device_interface.h"
 #include "dpctl_sycl_device_selector_interface.h"
+#include "dpctl_sycl_enum_types.h"
 #include "dpctl_sycl_platform_interface.h"
 #include "dpctl_sycl_queue_interface.h"
```

Comment:
`dpctl_sycl_queue_interface.h`
`dpctl_sycl_queue_manager.h`

These  old includes are no longer needed and should be removed.